src/js/controller.js

```diff
@@ -63,7 +63,7 @@ const timeout = function (s) {
 async function showRecipe() {
   try {
     const id = window.location.hash.slice(1);
-    console.log(id);
+    // console.log(id);
```

Comment:
commented logs are not needed in production, you should have that only while debugging something.

Reply:
@syedali042 How you identified this PR bro ?